src/__test__/components/data-exploration/embedding/Embedding.test.jsx

```diff
@@ -63,7 +63,7 @@ describe('Embedding', () => {
       },
     },
     cellInfo: {
-      cellName: 2,
+      cellId: 2,
```

Comment:
While you're making this change, can you also fix the relative import paths to use absolute import paths in this file?